scripts/download-vsix-from-circleci.js

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
+
+const shell = require('shelljs');
+shell.set('-e');
+shell.set('+v');
+
+const { getVsixName, getLocalPathForDownload } = require('./publish-utils');
+const CIRCLECI_API_URI = 'https://circleci.com/api/v1.1';
+const circleciToken = process.env['CIRCLECI_TOKEN'];
```

Comment:
We might want to include instructions on what these are and how to set them for new users that haven't set up circleci-to-vscode.